client/server/pages/index.js

```diff
@@ -102,7 +102,12 @@ function getDefaultContext( request, response, entrypoint = 'entry-main' ) {
 		response.cookie( 'country_code', geoIPCountryCode );
 	}
 
-	const cacheKey = `${ getNormalizedPath( request.path, request.query ) }:gdpr=${ showGdprBanner }`;
+	const cacheKey = getCacheKey( {
```

Comment:
This one's a bit weird... I tried to use `getCacheKey({...request, context:{}})`, but it didn't work. I think it might be in a weird format at this point of the bootstrapping process. 🤔

Reply:
It wasn't a good idea probably, because `request` will contain a bunch more things that we don't need here.